### PR TITLE
Updated all Zigbee configuration variable

### PR DESCRIPTION
**Pull request by klaasnicolaas**
Updated style of all Zigbee documentation to follow new configuration variables description.
Related to #6385.

**Description:**

**Pull request in [home-assistant](https://github.com/home-assistant/home-assistant) (if applicable):** home-assistant/home-assistant#<home-assistant PR number goes here>

## Checklist:

- [ ] Branch: `next` is for changes and new documentation that will go public with the next [home-assistant](https://github.com/home-assistant/home-assistant) release. Fixes, changes and adjustments for the current release should be created against `current`.
- [ ] The documentation follow the [standards][standards].

[standards]: https://home-assistant.io/developers/documentation/standards/

### CODE REVIEW

**Comment by klaasnicolaas** [comment]
What causes this error: Liquid Exception: (<unknown>): found character that cannot start any token while scanning for the next token at line 5 column 12 in /opt/build/repo/source/_components/zigbee.markdown

File: source/_components/zigbee.markdown [hunk 1]
```diff
@@ -31,8 +31,18 @@ A `zigbee` section must be present in the `configuration.yaml` file and contain
 zigbee:
 ```
 
-- **device** (*Optional*): The serial port to which the local ZigBee device is connected. Defaults to `/dev/ttyUSB0`
-- **baud** (*Optional*): The baud rate at which to communicate with the local ZigBee device. Defaults to `9600`
+{% configuration %}
+device:
+  description: The serial port to which the local ZigBee device is connected.
+  required: false
+  default: `/dev/ttyUSB0`
```
Comment (klaasnicolaas) [inline source/_components/zigbee.markdown]: Causes this a error?

File: source/_components/zigbee.markdown [hunk 2]
```diff
@@ -31,8 +31,18 @@ A `zigbee` section must be present in the `configuration.yaml` file and contain
...
+  type: string
+baud:
+  description: The baud rate at which to communicate with the local ZigBee device.
+  required: false
+  default: `9600`
```
Comment (klaasnicolaas) [inline source/_components/zigbee.markdown]: Same

File: source/_components/sensor.zigbee.markdown [hunk 3]
```diff
@@ -30,14 +30,29 @@ sensor:
...
+  type: integer
+address:
+  description: The long 64-bit address of the remote ZigBee device whose pin you would like to sample. Do not include this variable if you want to sample the local ZigBee device's pins.
+  required: false
+  type: string
+max_volts:
+  description: The maximum voltage which the input pin is able to read.
+  required: false
+  default: `1.2`
+  type: float
```
Comment (klaasnicolaas) [inline source/_components/sensor.zigbee.markdown:54]: Double or a float?

File: source/_components/zigbee.markdown (hunk 1 above)
Comment (WofWca) [reply source/_components/zigbee.markdown]: Yep. Can't start a value with ``` ` ```.

File: source/_components/sensor.zigbee.markdown (hunk 3 above)
Comment (WofWca) [reply source/_components/sensor.zigbee.markdown:54]: There's no `double` neither in Python nor in YAML. Standard doesn't say about `double` too, so I guess `float`.

Comment (klaasnicolaas) [reply source/_components/sensor.zigbee.markdown:54]: Previously I used a lot of java and there you had it, hence my question

**Comment by klaasnicolaas** [comment]
Have make some adjustments, hopefully it is good now

**Review by frenck — APPROVED** [review]

**Comment by frenck** [comment]
Thanks, m8! Looks awesome 👍 

Happy Hacktoberfest 🎉 

../Frenck

**Pull request merged**